app/javascript/courses/review/components/CoursesReview__Editor.res

```diff
@@ -999,6 +999,20 @@ let updateReviewer = (cb, send, reviewer) => {
   send(ShowGradesEditor)
 }
 
+let pageTitle = (number, submissionDetails) => {
+  let studentOrTeamName = switch SubmissionDetails.teamName(submissionDetails) {
+  | Some(teamName) => teamName
+  | None => {
+      let students = SubmissionDetails.students(submissionDetails)
+      Student.name(students[0])
+    }
+  }
+
+  `${t("submission")} ${number->Belt.Int.toString}  |  L${SubmissionDetails.levelNumber(
+      submissionDetails,
+    )}  |  ${studentOrTeamName}`
+}
```

Comment:
```suggestion
`${t("submission")} ${number->string_of_int} | ${t(
      "level_acronym",
    )}${SubmissionDetails.levelNumber(submissionDetails)} | ${studentOrTeamName}`
```

Reply:
I initially used this `string_of_int` for converting int to string. But I thought it was older syntax and used this `Belt.Int.toString`. Just to know any reason why we are still using this `string_of_int` , Is it for consistency across code.

Reply:
Mostly for consistency across the codebase.